### DISCUSSION TITLE
How to run certain steps when workflow is canceled?

### DISCUSSION THREAD

**Discussion by vitalykarasik** [community/community#25723] [Actions]
<p>I have a workflow which is launching self-hosted actions runners dynamically.<br>
As a last step I’m deleting these runners. But what if the workflow was canceled? Is it possible to run certain commands (steps) in this state?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The <a href="https://docs.github.com/en/actions/learn-github-actions/expressions#cancelled"><code>cancelled()</code> status check function</a> might help.</p>
